README.md

```diff
@@ -9,6 +9,24 @@ Here, you will find the following directories:
 ├── agent-lab               # Labs with exercises
 ```
 
-## Next step
+## Prerequisites
 
-To get started, check out the [Agent Lab introduction](./agent-lab/README.md).
+1. Github Account: Create an account or use your existing account for [Github](https://github.com)
+
+2. Microsoft Account: Create an account or use your existing account for [Microsoft](https://account.microsoft.com/account)
```

Comment:
maybe add clarifications on why this is needed?
```suggestion
2. Microsoft Account: Create an account or use your existing account for [Microsoft](https://account.microsoft.com/account). This will be required to access Azure resources.
```